core/domain/exp_services.py

```diff
@@ -385,14 +520,28 @@ def get_recently_published_exp_summaries(limit):
 def get_all_exploration_summaries():
     """Returns a dict with all exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        dict. A dict with all ExplorationSummary domain objects keyed by their
+        exploration id.
     """
     return _get_exploration_summaries_from_models(
         exp_models.ExpSummaryModel.get_all())
 
 
 # Methods for exporting states and explorations to other formats.
 def export_to_zip_file(exploration_id, version=None):
-    """Returns a ZIP archive of the exploration."""
+    """Returns a ZIP archive of the exploration.
+
+    Args:
+        exploration_id: str. The id of the exploration to export.
+        version: int or None. If provided, this indicates which version of
+            the exploration to export. Otherwise, the latest version of the
+            exploration is exported.
+
+    Returns:
+        zipfile. A ZIP archive of the exploration.
```

Comment:
str. The contents of the ZIP archive of the exploration (which can be subsequently converted into a zip file via zipfile.ZipFile()).

Note how this docstring does not say anything about what is actually done with the return value, so that the outside context does not "leak into" the function, but it provides context around how the return value could be used by clients (and this acts as a sort of spec for the person writing or modifying code within this function).